spec/requests/conversion_hosts_spec.rb

```diff
@@ -19,7 +23,7 @@
 
   context "resources" do
     it 'will show a conversion host with an appropriate role' do
-      conversion_host = FactoryBot.create(:conversion_host, :resource => FactoryBot.create(:vm))
+      conversion_host = FactoryBot.create(:conversion_host, :resource => FactoryBot.create(:vm_openstack))
```

Comment:
Where does the vm_openstack factory come from?  If it's in core, that's probably ok, otherwise this comes back to my pluggability question.

Reply:
It's from core: https://github.com/ManageIQ/manageiq/blob/master/spec/factories/vm_or_template.rb#L121

Reply:
Just saying: That factory should have been in the provider's repo. We should have all those factories in their own repo. Core is about concepts that transcend specific implementations (providers) and should operates only on core objects.